improver_tests/calibration/test_init.py

```diff
@@ -52,21 +52,16 @@
 from improver_tests import ImproverTest
 
 
-class Test_split_forecasts_and_truth(unittest.TestCase):
+def create_input_cubes(forecast_type):
```

Comment:
You could add types here e.g.

```suggestion
def create_input_cubes(forecast_type: str) -> Tuple[List, List, List, List]:
```

Reply:
Done